src/tools/ScenarioMeasurement/Startup/LinuxTraceSession.cs

```diff
@@ -1,10 +1,12 @@
 ﻿using System.Collections.Generic;
+using System;
 
 namespace ScenarioMeasurement
 {
     public class LinuxTraceSession : ITraceSession
     {
-        public string TraceFilePath { get { return perfCollect?.TraceFilePath; } }
+        public string TraceFilePath { 
+            get { return perfCollect?.TraceFilePath;} }
```

Comment:
Why this line break?

Reply:
In what way? I assume the null-conditional operator will return null if perfCollect is null. Do you mean we can remove the operator because we always initialize ```TraceFilePath``` in ```PerfCollect( )``` constructor?

Reply:
I believe he means why
```
public string TraceFilePath { 
            get { return perfCollect?.TraceFilePath;} }
```
instead of 
```
public string TraceFilePath { get { return perfCollect?.TraceFilePath;} }
```
or 
```
public string TraceFilePath
{
    get { return perfCollect?.TraceFilePath;}
}
```

Reply:
I literally read it as "Will this line break?"